feat: use the placeholder to show how to add a page break

joanise opened the pull request:
<!-- PR template: please provide enough information to guide your reviewers.
Please read Contributing.md before submitting a PR.  -->

### PR Goal? <!-- Explain the main objective of this PR. -->

One step towards addressing the fact that users don't discover how to add pages.

### Feedback sought? <!-- What should reviewers focus on in particular? -->

Does this feel OK? I'm just using the placeholder text, so it disappears when you start typing, but it hints at using newslines as separators.

(We still need to update the tour, though, this is just an extra bit that can already help some.)

### Priority? <!-- How soon would you like this PR reviewed, does it block other work? -->

Before ICLDC, if we decide to merge this.

### Tests added? <!-- Make sure your PR includes automated tests for your changes. -->

n/a

### How to test? <!-- Explain how reviewers should test this PR. -->

Load the preview and look at the textbox placeholder text.

### Confidence? <!-- How confident are you that these changes are ready to merge? -->

medium

### Version change? <!-- Do you think this PR should trigger a Major (Breaking Change)/Minor (New Feature)/patch (refactor/bug fix) version change? -->

no

<!-- Add any other relevant information here -->

joanise committed:
chore: bump caniuse-lite

joanise committed:
feat: use the placeholder to show how to add a page break (#389)

deltork commented:
The error generated is because the SRT format output from Studio-cli differs from the sample file we made for the test. One of the commits to CLI has slightly altered the SRT format output.
I just did a fresh download and found that the SRT file has two new lines at the end of the output see attachment (left is ref, right is new download)
<img  alt="image" src="https://github.com/user-attachments/assets/d51d8d5a-7ea3-409e-b1cc-90aaf65a528c" />

Solution: replace the `packages/studio-web/tests/fixtures/ref/readalong.srt` with new download or fix cli not to include the extra whitespace

deltork reviewed (changes requested):
Looks good!
Maybe fix the CLI SRT output or replace the reference file

joanise commented:
Interesting, the SRT output with changes to the newlines is the result of changing the version of the webvtt-py package in Studio.

In Studio's unit testing, I added strip() to both strings so tests would not depend on that detail. We'll have to either update the ref file here or do the same, so it's less flaky.

joanise commented:
I'll try using `trim()`, which appears to be the JavaScript equivalent of Python's `strip()`:
```
  await expect(
    replaceValuesWithZeroes(fileData.replace(/\r/g, "")).trim(),
    "file content should match reference data",
  ).toEqual(replaceValuesWithZeroes(refFileData).trim());
```

joanise committed:
fix(test): srt file comparison should tolerate newline diffs

The number of newlines at the end depends on the webvtt-py version in Studio,
and we want to be OK with any version.

joanise commented:
@deltork Thanks for the analysis on CI. It's fixed with those `.trim()` added on both the output and the reference. This way, the tests will work regardless of the version of Studio and its webvtt-py dependency you have installed.
